client/src/components/App/Content/InvestigationForm/TrackingRecommendation/TrackingRecommendationForm.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector } from 'react-redux';
+import { Grid , Collapse, Select, MenuItem, FormControl, InputLabel } from '@material-ui/core';
+
+import StoreStateType from 'redux/storeStateType';
+import TrackingRecommendation from 'models/TrackingRecommendation';
+import AlphanumericTextField from 'commons/NoContextElements/AlphanumericTextField';
+import { setTrackingRecommendation } from 'redux/Investigation/investigationActionCreators';
+
+import useStyles from './trackingRecommendationStyles';
+import { Option } from './trackingRecommendationTypes';
+import UseTrackingRecommendationForm from './useTrackingRecommendationForm';
+
+const otherSubReason = 99;
+const defaultTrackingReason = 0;
+const trackingOptions = [
```

Comment:
maybe create a file in models for that

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1426/commits/f0664bdc73f6c034c3e749827412aeef075b6225)